documentation/examples/04-events/01-inline-handlers/App.svelte

```diff
@@ -2,7 +2,7 @@
 	let m = { x: 0, y: 0 };
 </script>
 
-<div on:mousemove={(e) => (m = { x: e.clientX, y: e.clientY })}>
+<div on:mousemove={(e) => (m = { x: e.clientX, y: e.clientY })} role='button' tabindex='0'>
```

Comment:
Same comment - we should ignore this warning instead of adding role="button" (which won't help in this case)